it/stubs/CustomerInfoStub.scala

```diff
@@ -41,92 +41,74 @@ object CustomerInfoStub extends WireMockMethods {
       .thenReturn(status = INTERNAL_SERVER_ERROR, body = errorJson)
   }
 
-  private val customerInfo: JsValue = Json.parse(
-    """{
-      |  "organisationName" : "Cheapo Clothing Ltd",
-      |  "title" : "0001",
-      |  "firstName" : "Betty",
-      |  "middleName" : "Vatreturn",
-      |  "lastName" : "Jones",
-      |  "tradingName" : "Cheapo Clothing",
-      |  "mandationStatus" : "1",
-      |  "registrationReason" : "0001",
-      |  "effectiveRegistrationDate" : "2017-08-21",
-      |  "businessStartDate" : "2017-01-01",
-      |  "isPartialMigration" : false,
-      |  "PPOB":{
-      |    "address":{
-      |      "line1":"Bedrock Quarry",
-      |      "line2":"Bedrock",
-      |      "line3":"Graveldon",
-      |      "line4":"Graveldon",
-      |      "postCode":"GV2 4BB"
-      |    },
-      |    "contactDetails":{
-      |      "primaryPhoneNumber":"01632 982028",
-      |      "mobileNumber":"07700 900018",
-      |      "emailAddress":"bettylucknexttime@gmail.com"
-      |    }
-      |  },
-      |  "correspondenceContactDetails":{
-      |    "address":{
-      |      "line1":"13 Pebble Lane",
-      |      "line2":"Bedrock",
-      |      "line3":"Graveldon",
-      |      "line4":"Graveldon",
-      |      "postCode":"GV13 4BJ"
-      |    },
-      |    "contactDetails":{
-      |      "primaryPhoneNumber":"01632 960026",
-      |      "mobileNumber":"07700 900018",
-      |      "emailAddress":"bettylucknexttime@gmail.com"
-      |    }
-      |  }
-      |}""".stripMargin
+  val customerInfoJsonMax: JsValue = Json.obj(
```

Comment:
Is it possible to get this from test/common/TestJson.scala instead of repeating here?

Reply:
It picks up the import but then errors when the tests run. I'm not sure if it fails because it's in a different test package. I tried using `_root_` in front of the import too but got the same issue. If you know how to do it then let me know.

Reply:
I've had a go too but no dice! It's fine as is 👍